src/log-pcap.c

```diff
@@ -171,6 +168,11 @@ void TmModulePcapLogRegister(void)
     (prof).total += (UtilCpuGetTicks() - pcaplog_profile_ticks); \
     (prof).cnt++
 
+static int PcapLogCondition(ThreadVars *tv, const Packet *p)
+{
+    return TRUE;
```

Comment:
this seems too broad. E.g. pseudo packets should be excluded. Not every condition here https://github.com/inliniac/suricata/blob/master/src/log-pcap.c#L358 can be handled in this function though.